### PR TITLE
REMOVE: Serving Drinks

### PR DESCRIPTION
**Pull request by nickswalker**
## Description

Towards #685, we agreed to cut this task during the last meeting #694

### CODE REVIEW

File: Rulebook.tex [hunk 1]
```diff
@@ -70,10 +70,8 @@ \section*{Housekeeper tasks}
 
 \section*{Party Host tasks}
 \testtocentry{test:carry-my-luggage}
-\testtocentry{test:farewell}
```
Comment (kyordhel) [inline Rulebook.tex]: Why is Farewell here?

File: Rulebook.tex [hunk 2]
```diff
@@ -103,12 +99,9 @@ \chapter{Tests in Stage II}
 \section*{Housekeeper tasks}
 \testtocentry{test:clean-the-table}
 \testtocentry{test:egpsr}
-\testtocentry{test:find-my-disk}
```
Comment (kyordhel) [inline Rulebook.tex]: Nope

File: Rulebook.tex [hunk 3]
```diff
@@ -103,12 +99,9 @@ \chapter{Tests in Stage II}
...
-\testtocentry{test:set-the-table}
```
Comment (kyordhel) [inline Rulebook.tex]: Nope

**Comment by nickswalker** [comment]
All of these comments are on a commit that's meant to be merged on #695. Please check again once that commit is on master.

**Pull request merged**